docs/tdvt.md

```diff
@@ -544,3 +544,41 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+## Running TDVT test with tabquerytoolsrv.exe
+You can run a particular TDVT test with the help of a test setup file using tabquerytoolsrv.exe. `tabquerytoolsrv.exe` is located in `C:\Program Files\Tableau\Tableau Server\packages\bin.<version_code>\tabquerytoolsrv.exe` for Windows and `/opt/tableau/tableau_server/packages/bin.<version_code>/tabquerytoolsrv` for Linux. 
+ Before being able to run TDVT tests using tabquerytoolsrv make sure that: 
+   - Your instance of the tableau server is running 
+   - Your server license has been activated 
+Once the above steps are verified you should be able to run TDVT tests using tabquerytoolsrv by running the following command:
+
+__For Windows__
+```
+ tabquerytoolsrv.exe --query-file-list D:\test\tests.txt -d D:\path_to_tds\cast_calcs.db.tds --combined --password-file D:\path_to_tds\db.password --output-dir D:\test -DLogDir=D:\test -DOverride=ProtocolServerNewLog -DLogicalQueryRewriteDisable=Funcall:RewriteConstantFuncall -DInMemoryLogicalCacheDisable -DParamFile="C:\ProgramData\Tableau\Tableau Server\data\tabsvc\config\minerva_0.<version_code>\workgroup.yml,C:\ProgramData\Tableau\Tableau Server\data\tabsvc\config\minerva_0.<version_code>\minerva.yml"
+Attempting to run query...
+Run query successful! Check output file
+```
+
+__For Linux:__
```

Comment:
Is this content below like a troubleshooting section? As long as TAB_CLI_EXE_LINUX or TAB_CLI_EXE_X64 is configured for tabquerytoolsrv, doesn't TDVT work the same as documented?

Reply:
I tried adding tabquerytoolsrv in `tdvt_override.ini` but this did not work. 
```
TAB_CLI_EXE_X64 = C:\Program Files\Tableau\Tableau Server\packages\bin.near.22.0802.1408\tabquerytoolsrv.exe
TAB_CLI_EXE_MAC = ../../tableau-1.3/build/Release/tabquerycli
TAB_CLI_EXE_LINUX = ../../tableau-1.3/build/Release-x64/tabquerycli
```
The TDVT logs specify that the prams file are not present
```
2022-08-23 13:49:47,223 Thread-[5] CalledProcessError: Return code: 1 ParamFile not specified.

2022-08-23 13:49:47,224 Thread-[6] CalledProcessError: Return code: 1 ParamFile not specified.

2022-08-23 13:49:47,224 Thread-[5] Command line output:ParamFile not specified.

2022-08-23 13:49:47,224 Thread-[6] Command line output:ParamFile not specified.

2022-08-23 13:49:47,225 Thread-[1] CalledProcessError: Return code: 1 ParamFile not specified.
```

Reply:
Looks like we might need to do some work to add the param files in tdvt_override.ini file and logic to add the command for if tabquerytoolsrv is being used.

Reply:
Looks like the command that TDVT calls is the same but without the params file 
```
2022-08-23 13:49:45,887 Thread-[6]  calling C:\Program Files\Tableau\Tableau Server\packages\bin.near.22.0802.1408\tabquerytoolsrv.exe --expression-file-list C:\Users\STRIPA~1\AppData\Local\Temp\66c8231c15251d74fd0c4686cda2cd7bf2df7a16c4e0037b69a2a94e3elozhlk\expression_lod_postgres\tests.txt -d D:\dev\near\tableau-tests\tdvt\tds\cast_calcs.postgres.tds --combined --password-file D:\dev\near\tableau-tests\tdvt\tds\postgres.password --output-dir C:\Users\STRIPA~1\AppData\Local\Temp\66c8231c15251d74fd0c4686cda2cd7bf2df7a16c4e0037b69a2a94e3elozhlk -DLogDir=C:\Users\STRIPA~1\AppData\Local\Temp\66c8231c15251d74fd0c4686cda2cd7bf2df7a16c4e0037b69a2a94e3elozhlk\expression_lod_postgres -DOverride=ProtocolServerNewLog -DLogicalQueryRewriteDisable=Funcall:RewriteConstantFuncall -DInMemoryLogicalCacheDisable
```

Reply:
I think that getting this flow working is necessary for TDVT on Linux to be considered done, so the documentation may be premature. May need to schedule more work to update TDVT to do this work for tabquerysrv (while also not breaking our existing Linux testing internally, which uses tabquerytool.) Maybe adding TAB_CLI_LINUX_SRV that will add the params file?

Reply:
Looks like using CommandLineOverride to add the -DParaFile works. Updating the doc to reflect that.